applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp

```diff
@@ -138,8 +138,8 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
 
     std::string Info() const override
     {
-        return "U-Pw Base class Element #" + std::to_string(Id()) +
-               "\nConstitutive law: " + mConstitutiveLawVector[0]->Info();
+        const std::string constitutive_info = !mConstitutiveLawVector.empty() ? mConstitutiveLawVector[0]->Info() : "not defined";
```

Comment:
Thank you for making the proposed change, could you also do this for the other 'Info' functions you fixed?

Reply:
OK. I'll try to find the time to do it.

Reply:
Ok, let me know if you'd like me to do these changes, happy to help 👍

Reply:
Sure. Feel free to pirate the PR 😄 🏴‍☠️